lib/pkt-line.js

```diff
@@ -16,7 +16,7 @@ function deframer(emit) {
   var more = true;
 
   return function (item) {
-
+    var doUndefined = false;
     // Forward the EOS marker
     if (item === undefined) return emit();
```

Comment:
This line is probably called in the tcp transport, but never in the HTTP transport.